src/Command/ImportMappingDoctrineCommand.php

```diff
@@ -1,166 +0,0 @@
-<?php
-
-namespace Doctrine\Bundle\DoctrineBundle\Command;
-
-use Doctrine\ORM\Mapping\ClassMetadata;
-use Doctrine\ORM\Mapping\Driver\DatabaseDriver;
-use Doctrine\ORM\Tools\Console\MetadataFilter;
-use Doctrine\ORM\Tools\DisconnectedClassMetadataFactory;
-use Doctrine\ORM\Tools\Export\ClassMetadataExporter;
-use Doctrine\Persistence\ManagerRegistry;
-use InvalidArgumentException;
-use Symfony\Component\Console\Input\InputArgument;
-use Symfony\Component\Console\Input\InputInterface;
-use Symfony\Component\Console\Input\InputOption;
-use Symfony\Component\Console\Output\OutputInterface;
-
-use function assert;
-use function chmod;
-use function dirname;
-use function file_put_contents;
-use function is_dir;
-use function mkdir;
-use function sprintf;
-use function str_replace;
-
-/**
- * Import Doctrine ORM metadata mapping information from an existing database.
- *
- * @deprecated
- *
- * @final
- */
-class ImportMappingDoctrineCommand extends DoctrineCommand
-{
-    /** @param string[] $bundles */
-    public function __construct(
-        ManagerRegistry $doctrine,
-        private readonly array $bundles,
-    ) {
-        parent::__construct($doctrine);
-    }
-
-    protected function configure(): void
-    {
-        $this
-            ->setName('doctrine:mapping:import')
```

Comment:
we also need to remove the service definition then

https://github.com/doctrine/DoctrineBundle/blob/5ed195e326f4a5a35fa507968f8b197e24db07b3/config/orm.php#L326-L331